### PR TITLE
Bookstore: Project setup with components

### PR DESCRIPTION
**Pull request by keebot-png**
## In this project I performed the following actions in line with project specifications:
- Fix linter errors.
- Include minimum styling.
- Set up routing using React Router.
- Created the React part of the app.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,285 +1,56 @@
...
+- If it suits you, on your local computer, create a directory in your preferred location (e.g in the Desktop or Document directory)
 
-```sh
-  rails server
-```
---->
+## Install
 
-### Run tests
+- To clone the project on your machine: Type https://github.com/Jaymelfah/Bookstore.git.
```
Comment (elmar8287) [inline README.md]: - [ ] Please, consider fixing this link not related to you. I think it should be `https://github.com/keebot-png/bookstore-1.git` with `git clone` command. A full command to clone the project should be `git clone https://github.com/keebot-png/bookstore-1.git`

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, @keebot-png.

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
- Amazing considering the styling 🥇 
- Good PR description with the commits history 👍 

## Required Changes ♻️

- [ ] Check the comments under the review.

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
### Please, remember to tag me (@elmar8287) in your question so I can receive the notification 👍 
giphy

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (keebot-png) [reply README.md]: @elmar8287 Thanks for noting this. I needed to copy a decent readme file for this project as the newer template is very lengthy and will take some time to adapt.

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @keebot-png, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**